upgrade_tests/test_scenarios.py

```diff
@@ -0,0 +1,251 @@
+# -*- encoding: utf-8 -*-
+"""Test for Upgrade Scenario's
+
+:Requirement: Satellite upgrade
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import os
+import time
+
+from automation_tools.satellite6 import hammer
+from upgrade.helpers.docker import (
+    docker_execute_command,
+    refresh_subscriptions_on_docker_clients
+)
+from upgrade_tests.helpers.scenarios_utils import (
+    create_dict,
+    dockerize,
+    get_entity_data,
+    get_latest_repo_version,
+    get_satellite_host,
+    rpm1,
+    rpm2
+)
+from fabric.api import execute, run
+from fauxfactory import gen_alpha
+from unittest2.case import TestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+
+
+class ScenarioBug1429201(TestCase):
+    """This Class will server as a whole scenario with pre-upgrade and
+    post-upgrade test-case
+    """
+    prd_name = gen_alpha()
```

Comment:
gen_alpha() generated strings will trouble us during debugging this scenario.

Suppose you have 4 scenarios and you have created 5 products during these scenarios execution, then it would be very difficult to link up that product with the scenario. We should have some trace of scenario with the each entity that we are creating during upgrade.

with that saying, lets change to this:
```
prd_name = 'scenario1-'+gen_alpha()
```
So that debugging will become very easy and we need not dive in all entities to get their links with scenarios.